_backend/config/settings.py

```diff
@@ -112,15 +113,21 @@
 
 if DEBUG:
     DATABASES = {
-            'default': {
-                'ENGINE':DB_ENGINE,
-                'NAME':DB_DEVELOPER_NAME,
-                'USER':DB_DEVELOPER_USERNAME,
-                'PASSWORD':DB_DEVELOPER_PASSWORD,
-                'HOST':DB_DEVELOPER_HOST,
-                'PORT':DB_DEVELOPER_PORT,
-                }
-            }
+        'default': {
+            'ENGINE': 'django.db.backends.sqlite3',
+            'NAME': BASE_DIR / 'db.sqlite3',
+        }
+    }
+    # DATABASES = {
```

Comment:
Is this to uncomment for developers who want to debug?

Reply:
I took down the Postgres server, so if you want to run the app you need a database: Hence the default; We can delete or if I pull up the postgress server, we can uncomment.